model/historical_test_data.go

```diff
@@ -57,3 +135,10 @@ func (i *HistoricalTestDataInfo) validate() error {
 
 	return catcher.Resolve()
 }
+
+func (i *HistoricalTestDataInfo) getPath(artifactType PailType) string {
+	if artifactType == PailLocal {
+		return filepath.Join(i.Project, i.Variant, i.TaskName, i.TestName, i.RequestType, fmt.Sprintf("%d", i.Date.Unix()))
```

Comment:
shouldn't we make both of these strings.Join([]string{<list>}, "/")

Reply:
I used filepath because for windows the slash is the other way

Reply:
oh, I see.